lua/telescope/make_entry.lua

```diff
@@ -925,6 +925,10 @@ function make_entry.gen_from_ctags(opts)
     table.insert(display_items, 1, { width = 30 })
   end
 
+  if opts.fname_width then
+    table.insert(display_items, 1, { width = opts.fname_width })
+  end
```

Comment:
Did you test this? I think this ^ will not work. I only meant you dont want `opts.fname_width or 30` in line 925 because that is the setting for the show_line option. Change in line 921 was good. (even tho i prefer `vim.F.if_nil` over `or` here

Reply:
Okay, both changes seemed to work initially. So, I did another bout of thorough testing. I think there is a slightly bigger issue here. Since width is prepended to the display items, different widths do not work for all combinations of options. So far the path and show_line work well only because the widths are the same by default (30).  I'll try to figure out how to set the correct width for the correct component.

To explain the issue better: {show_line = false, fname_width = 10} does the expected thing of making file paths to 10 wide. But, {show_line = true, fname_width = 10} makes the "Line" component to be 10 wide instead of file path because it gets pushed down in the display_items table.

Reply:
Thats why i said this change above isnt good and we were looking for your initial change

```lua
  if not hidden then
    table.insert(display_items, 1, { width = vim.F.if_nil(opts.fname_width, 30) })
  end
```